docs/resources/user_password_policy_attachment.md

```diff
@@ -0,0 +1,27 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "snowflake_user_password_policy_attachment Resource - terraform-provider-snowflake"
+subcategory: ""
+description: |-
+  Specifies the password policy to use for the current account. To set the password policy of a different account, use a provider alias.
+---
+
+# snowflake_user_password_policy_attachment (Resource)
+
+Specifies the password policy to use for the current account. To set the password policy of a different account, use a provider alias.
+
```

Comment:
Please provide example usage. You can do it by creating a directory s`nowflake_user_password_policy_attachment` inside `examples/resources` (you can check the one for account password policy - `snowflake_account_password_policy_attachment`). You will have to re-run `make docs` or `make pre-push` after that.

Reply:
The same with import example.

Reply:
Done!